public/app/plugins/datasource/zipkin/plugin.json

```diff
@@ -26,6 +26,10 @@
         "name": "Learn more",
         "url": "https://zipkin.io"
       }
-    ]
+    ],
+    "version": "%VERSION%"
+  },
+  "dependencies": {
+    "grafanaDependency": ">=10.3.0-0"
```

Comment:
Am I wrong or this should be 10.4.x? 🤔

Reply:
it depends, if the plugin depends on any core feature that was included in 10.4 then yes, if not, external core plugins are supported since 10.3

Reply:
Yeah this was intended because AFAIK the plugin does not depend on features of 10.4.